src/ipaperftest/core/constants.py

```diff
@@ -545,3 +562,45 @@ def getLevelName(level):
       line: selinux_provider = none
       insertafter: id_provider = ipa
 """
+
+ANSIBLE_GROUPSIZETEST_SERVER_CONFIG_PLAYBOOK = """
+---
+- name: Configure server before execution
+  hosts: ipaserver
+  become: yes
+  tasks:
+    - copy:
+        dest: /etc/ipa/server.conf
+        content: |
+          [global]
+          debug=True
+    - command:
+        cmd: "systemctl restart httpd"
+    - synchronize:
+        src: "{{{{ item }}}}"
+        dest: "/root"
+        mode: push
+        use_ssh_args: yes
+      with_items:
+        - create-test-data.py
+        - batch.py
+    - package:
+        name: python3-pip
+    - package:
+        name: time
```

Comment:
You can install multiple packages with a single call to the package module using a list instead: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/package_module.html